docs/flow/tutorials/assets/flow_configs/multiple-inputs.flow

```diff
@@ -0,0 +1,35 @@
+prometheus.scrape "first" {
+    targets = [{"__address__" = "localhost:12345"}]
+    forward_to = [prometheus.relabel.cool.receiver,prometheus.relabel.not_cool.receiver]
+}
+
+prometheus.scrape "second" {
+    targets = [{"__address__" = "localhost:12345"}]
+    forward_to = [prometheus.relabel.cool.receiver,prometheus.relabel.not_cool.receiver]
+}
+
+prometheus.relabel "cool" {
+    metric_relabel_config {
+        source_labels = ["__name__"]
+        regex = "(.+)"
+        replacement = "${1}_cool"
```

Comment:
From my testing, the `${}` is required to work.

Reply:
That can't be true, the [default is `$1`](https://github.com/prometheus/prometheus/blob/35d6813963b3f82c5d07f3602cc5e6decd12f53e/model/relabel/relabel.go#L34) and not `${1}`.

In the past, we also asked users to use `$1` over `${1}` because `${1}` was initially getting treated as an environment variable when -config.expand-env was passed.

Reply:
Is something on the River side breaking the `$1` format?

Reply:
After a discussion offline, `$1_cool` seems to be treated as a single capture group named `1_cool`, probably because `_` is treated as a word character, so `${1}` must be used here.

`$1` is only valid when followed by a non-word character like `/`. Resolving, but let's keep an eye on this since it might cause problems in the future as River expands.

Reply:
Circle back around looks to be a specific interaction with `_` , `${1}` seems to always work and `$1` works most of the time.